app-editors/featherpad/featherpad-1.1.0.ebuild

```diff
@@ -8,21 +8,30 @@ inherit xdg cmake
 DESCRIPTION="Lightweight Qt5 Plain-Text Editor for Linux"
 HOMEPAGE="https://github.com/tsujan/FeatherPad"
 SRC_URI="https://github.com/tsujan/FeatherPad/archive/V${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/FeatherPad-${PV}"
 
 LICENSE="GPL-3+"
 SLOT="0"
-KEYWORDS="~amd64 ~x86"
+KEYWORDS="~amd64 ~riscv ~x86"
+IUSE="+X"
```

Comment:
Out of curiosity, what does `-X` do here since it still pulls a lot of Qt stuff, including Qtgui. Is it supposed to be pure wayland-only option?

Reply:
This is almost turning into a FAQ -  see metadata.xml.